scipy/spatial/transform/rotation.pyx

```diff
@@ -1059,28 +1178,35 @@ def as_rotvec(self):
         (2, 3)
 
         """
-        quat = self._quat.copy()
-        # w > 0 to ensure 0 <= angle <= pi
-        quat[quat[:, 3] < 0] *= -1
-
-        angle = 2 * np.arctan2(np.linalg.norm(quat[:, :3], axis=1), quat[:, 3])
+        cdef Py_ssize_t num_rotations = len(self)
+        cdef double angle, scale, angle2
+        cdef double[:,:] rotvec = _empty2(num_rotations, 3)
+        cdef double[:] quat
+
+        for ind in range(num_rotations):
+            if self._quat[ind, 3] < 0: # w > 0 to ensure 0 <= angle <= pi
```

Comment:
I believe PEP8 recommendation is to use at least 2 spaces before the inline comment. Very minor, but still.

Reply:
Fix some that I noticed